web/css/spinner-button.css

```diff
@@ -0,0 +1,39 @@
+.spinnerButton {
+  display: inline-block;
+  background: black;
+  color: white;
+  position: relative;
+  transition: padding-right .3s ease;
+  font-weight: 700;
+}
+
+.spinnerButton.submitting {
```

Comment:
this should be `.spinning` as the class used in the spinner button is `spinning`. IMHO this makes sense as we might not always need it for submitting, but also other actions

Reply:
Ok, you asked me above to change .spinning for .submitting ;) I will put back .spinning then